plugins/karma_store.py

```diff
@@ -13,7 +13,12 @@
 ]
 
 
+@utils.register_command(name='buy', channels=['#karma-store'])
```

Comment:
what does the '#' mean? why sometimes we its just 'bot' and sometimes '#bot'?
I think it should be regex

Reply:
# means the exact name
without it means the string must be only part of the channel name

Reply:
in regex it would be just '^bot$' for the exact name or 'bot' for part of the channel name